run-tests.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT_NUMBER || 8000;
+const spawn = require('child_process').spawn;
+const platform = require("os").platform()
+
+// We need to change the command path based on the platform they're using
+const cmd = /^win/.test(platform)
+  ? `${process.cwd()}\\node_modules\\.bin\\mocha.cmd`
+  : `mocha`
+
+app.use(express.static(__dirname))
```

Comment:
You can use the `http-server` package here to serve stuff, same as we were doing in the bash script and like we do in the SDKs that start the server from JavaScript (e.g. https://github.com/percy/example-percy-puppeteer-standalone/blob/master/puppeteer_snapshots.js).

It's a little bit less work that using `express`:
```javascript
  const server = httpServer.createServer()
  const PORT = 8000
  server.listen(PORT)

[...do stuff..]

server.close()
```
(You can pass `__dirname` into the `createServer` call if you need to.)

Don't feel strongly about this, so fine to leave it using `express`.

Reply:
Oh damn! I briefly looked at their docs and couldn't find anything about creating a server in node (only CLI options) so I fell back to what is familiar. 

I also don't have any preferences 😂 But I might redo it so we can have consistency